src/components/Message.css

```diff
@@ -1,9 +1,9 @@
 .message {
   display: flex;
   margin-bottom: 0.25rem;
-  padding: 0.25rem;
-
-  line-height: 1.5rem;
+  padding: 1px;
```

Comment:
Any reason for `1px` padding vs no padding or `0rem`? I try to avoid using `px` except for border sizes.

Reply:
Oh yeah, its basically a border, but I will remove it so we can stick to `rem`s.